src/components/Scene.java

```diff
@@ -24,11 +24,12 @@ public void parse() {
             switch (tokenizer.checkNext()) {
                 case "text":
                     tokenizer.getAndCheckNext("text");
-                    while (!Main.literals.contains(tokenizer.checkNext())) {
+                    while (!Main.literals.contains(tokenizer.checkNext()) && !tokenizer.checkNext().equals("NO_MORE_TOKENS")) {
                         texts.add(tokenizer.getNext());
                     }
                     break;
                 case "choice":
+                    tokenizer.getAndCheckNext("choice");
```

Comment:
Make sure we get the choice, or else it'll go into creating a Choice object, and then check the next token as "choice", which doesn't match the regex we want